### PR TITLE
 [fix bug 1138379] Display up to three locales in admin. 

### PR DESCRIPTION
**Pull request by glogiotatidis**

### CODE REVIEW

**Comment by mozilla-travis** [comment]
```
snippets/urls.py:14:1: E402 module level import not at top of file
snippets/urls.py:24:5: E128 continuation line under-indented for visual indent
snippets/urls.py:31:1: E266 too many leading '#' for block comment
snippets/urls.py:40:9: E128 continuation line under-indented for visual indent
snippets/settings/docker.py:13:22: E203 whitespace before ':'
snippets/settings/docker.py:14:26: E203 whitespace before ':'
snippets/settings/docker.py:27:1: E402 module level import not at top of file
snippets/settings/__init__.py:4:1: F403 'from base import *' used; unable to detect undefined names
snippets/settings/__init__.py:8:5: F403 'from travis import *' used; unable to detect undefined names
snippets/settings/__init__.py:10:5: F403 'from docker import *' used; unable to detect undefined names
snippets/settings/__init__.py:13:9: F403 'from local import *' used; unable to detect undefined names
snippets/settings/__init__.py:21:9: F403 'from test import *' used; unable to detect undefined names
snippets/settings/travis.py:13:22: E203 whitespace before ':'
snippets/settings/travis.py:14:26: E203 whitespace before ':'
snippets/settings/travis.py:27:1: E402 module level import not at top of file
snippets/base/managers.py:50:101: E501 line too long (113 > 100 characters)
snippets/settings/base.py:4:1: F403 'from funfactory.settings_base import *' used; unable to detect undefined names
snippets/base/tests/test_forms.py:9:101: E501 line too long (101 > 100 characters)
snippets/base/views.py:27:1: E731 do not assign a lambda expression, use a def
snippets/base/admin.py:58:1: E303 too many blank lines (3)
snippets/base/admin.py:66:1: E303 too many blank lines (3)
snippets/base/admin.py:130:21: E222 multiple spaces after operator
snippets/base/admin.py:169:101: E501 line too long (112 > 100 characters)
snippets/base/admin.py:200:9: F841 local variable 'text' is assigned to but never used
snippets/base/admin.py:258:37: E713 test for membership should be 'not in'
snippets/base/tests/test_models.py:19:1: E302 expected 2 blank lines, found 1
snippets/base/models.py:163:1: E303 too many blank lines (3)
snippets/base/models.py:359:101: E501 line too long (103 > 100 characters)

```

**Comment by glogiotatidis** [comment]
@pmclanahan r?

**Comment by glogiotatidis** [comment]
@pmclanahan do you have the time to take a look? This and #99 are small changes and I thought that they are good opportunities to get your dev env up and running. thanks!

**Comment by pmclanahan** [comment]
DOH! Sorry. Missed this in my email. Looking now.

What is @mozilla-travis trying to tell us? Also what is @mozilla-travis ?

File: snippets/base/admin.py [hunk 1]
```diff
@@ -127,7 +127,11 @@ def change_view(self, request, *args, **kwargs):
         return super(BaseSnippetAdmin, self).change_view(request, *args, **kwargs)
 
     def locales(self, obj):
-        return ', '.join([locale.get_locale_display() for locale in obj.locale_set.all()])
+        locale_set =  obj.locale_set.all()[:4]
```
Comment (pmclanahan) [inline snippets/base/admin.py]: I know it'd be an extra query, but I think it'd be useful to indicate how many more.

```python
num_locales = obj.locale_set.count()
locale_set = obj.locale_set.all()[:3]
active_locales = ', '.join([locale.get_locale_display() for locale in locale_set])
if num_locales > 3:
    active_locales += ', and {0} more.'.format(num_locales - 3)

return active_locales
```

**Comment by pmclanahan** [comment]
My comment is not a blocker. r+ but feel free to update with my suggestion if you agree.

**Comment by glogiotatidis** [comment]
> What is @mozilla-travis trying to tell us? Also what is @mozilla-travis ?

mozilla-travis posts flake8 violations as github comments. After we merged #100 the flake8 comments should be 0. see also https://github.com/mozilla/snippets-service/blob/master/bin/comment-on-pr.py

**Comment by glogiotatidis** [comment]
Updated with your suggestions and merging. Thanks @pmclanahan  :)

**Pull request merged**